lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -715,6 +719,56 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
                 , expectField (#assets . #total . #getApiT) (`shouldNotBe` TokenMap.empty)
                 ]
 
+    it "TRANS_ASSETS_CREATE_02c - Send SeaHorses" $ \ctx -> runResourceT $ do
+            -- Notes on the style of this test:
+            -- - By doing the minting here it is easier to control, and tweak
+            -- the values.
+            -- - The current _mintSeaHorseAssets cannot be called concurrently
+            -- - By grouping the setup of multiple wallets in a single test, we
+            -- gain some time.
+
+            -- 1. Setup by minting funds
+            let assetsPerAddrScenarios = [64 .. 70]
+            sourceWallets <- forM assetsPerAddrScenarios $ \nAssetsPerAddr -> do
+                wSrc <- fixtureWallet ctx
+                srcAddrs <-
+                    map (getApiT . fst . view #id) <$> listAddresses @n ctx wSrc
+                liftIO $ _mintSeaHorseAssets ctx nAssetsPerAddr (take 2 srcAddrs)
+                return (wSrc, nAssetsPerAddr)
+            wDest <- emptyWallet ctx
+            destAddr <- head . map (view #id) <$> listAddresses @n ctx wDest
+            waitForTxImmutability ctx
+
+            -- 2. Try spending from each wallet, and record the response.
+            responses <- forM sourceWallets $ \(wSrc, nPerAddr) -> do
+                let seaHorses = map $ \ix ->
+                        (( toText seaHorsePolicyId
+                        , toText $ seaHorseTokenName ix)
+                        , 1)
+                payload <- mkTxPayloadMA @n
+                    destAddr
+                    0
+                    (seaHorses [1, nPerAddr * 2])
```

Comment:
Magic `2` here. I did consider removing, but you'd need to do more than `let nAddrs = 2` to have it actually make sense. E.g. choose the relevant indexes in the setup phase. Wasn't sure if worth it.